src/hooks/useCds/translate.js

```diff
@@ -231,12 +232,83 @@ const snomedPregnancyCare = {
   'display': 'Antenatal care (regime/therapy)'
 };
 
+const immunosuppressedObservation = {
+  label: 'Immunosuppressed',
+  code: {
+    coding: [
+      {
+       system: SCT_URL,
+        code: '370388006',
+        display: 'Patient immunocompromised (finding)'
+      }
+    ]
+  }
+};
+
+const pregnantObservation = {
+  label: 'Pregnant',
+  code: {
+    coding: [
+      {
+        system: SCT_URL,
+        code: '77386006',
+        display: 'Patient currently pregnant (finding)'
+      }
+    ]
+  }
+};
+
+const pregnantConcernedObservation = {
+  label: 'PregnantConcerned',
+  code: {
```

Comment:
Add text values to the `code` and `valueCodeableConcept` of the manually created Observations, so that if we do decide to display them somewhere there will be a human-readable value to display. For example, you can see in the current configuration that there is no human-readable name or value for each of the manually added Observations:


<img width="885" alt="Screenshot 2024-02-25 at 4 52 21 PM" src="https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/assets/51176978/d9f7faa7-fd5f-4937-b0bc-e4393ad87bb8">

Reply:
Fixed for Future Pregnancy Concerns, Symptomatic and Immunosuppressed.

Reply:
I did some digging, and it looks like the Pregnancy data is not being displayed when the "Pregnant" switch is toggled, because there is no status field in the Observation we create. https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/DashboardLibrary.cql#L513-L520 checks for the status of the Pregnancy Observation, so that a patient will not be considered pregnant if there is no status, and furthermore, the pregnancy will not be included in the patient history without a status.

Reply:
Status was added in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/32/commits/e3a32e1bb87dc846f6d9e91617335b795917fb56.